pkg/logger/logger.go

```diff
@@ -149,28 +120,11 @@ func (e *Entry) Log(level logrus.Level, msg string) {
 
 	domain, haveDomain := e.entry.Data["domain"]
 
-	if haveDomain && level == logrus.DebugLevel {
-		now := time.Now()
-		expiration, ok := domains.Load(domain)
-
-		switch {
-		case ok && now.Before(expiration.(time.Time)):
-			// The domain is listed in the debug domains and the ttl is valid, use the debuglogger
-			// to debug
-			debugLogger.WithFields(e.entry.Data).Log(logrus.DebugLevel, msg)
-			return
-
-		case ok && now.After(expiration.(time.Time)):
-			// The domain is listed in the debug domains but the ttl is no more valid, remove
-			// it from the list and do nothing to the log.
-			err := RemoveDebugDomain(domain.(string))
-			if err != nil {
-				e.entry.Logger.Errorf("failed to remove debug domain %q: %s", domain, err)
-			}
-
-		default:
-			// Log with the default logger
-		}
+	if haveDomain && level == logrus.DebugLevel && debugger.ExpiresAt(domain.(string)) != nil {
```

Comment:
Why not use `DebugExpiration(domain)` here?

Reply:
DebuExpiration only global function calling `debugger.ExpiresAt` and this method is already called inside the condition.